crates/holochain_sqlite/src/sql/conductor/select_valid_unrestricted_cap_grant.sql

```diff
@@ -1,18 +1,20 @@
 SELECT
-  blob
+  Entry.blob
 FROM
   Entry
+  INNER JOIN Action ON Action.author = ?2
+  AND Action.entry_hash = Entry.hash
 WHERE
   access_type = ?1
   AND (
     SELECT
-      COUNT(Action.hash)
+      COUNT(UpdateActions.hash)
     FROM
-      Action
+      Action as UpdateActions
```

Comment:
For my understanding, why rename Action to UpdateActions here?

Reply:
Ahh I think the PR description explains it, this is looking for updates/deletes that affect this cap grant?

I wonder if that's worth a comment left behind to help future readers follow the query?

Reply:
> Ahh I think the PR description explains it, this is looking for updates/deletes that affect this cap grant?

Correct.

> I wonder if that's worth a comment left behind to help future readers follow the query?

Yes, I'll do that!